source/features/warn-pr-from-master.tsx

```diff
@@ -27,14 +28,24 @@ async function init(): Promise<false | void> {
 	);
 }
 
+function init(signal: AbortSignal): void {
+	observe('.is-cross-repo', warnPR, {signal});
+}
+
 void features.add(import.meta.url, {
 	include: [
 		pageDetect.isCompare,
 	],
 	exclude: [
 		pageDetect.isBlank,
 	],
```

Comment:
This can also be dropped, see https://github.com/refined-github/refined-github/issues/7000#issuecomment-2287762931

```suggestion
```